content/applications/services/helpdesk/timesheet_and_invoice/reinvoice_from_project.rst

```diff
@@ -2,35 +2,83 @@
 Invoice Time Spent on Tickets (Postpaid Support Services)
 =========================================================
 
-Directly pull the billable time you have tracked on your helpdesk tickets into sales orders and
-invoices through a project task. It gives you more control over what you charge your client, and it
-is more efficient.
+The :guilabel:`Timesheet on Ticket` integration provides more control and transparency over how
+clients are charged, and what they're specifically billed for.
+
+In Odoo, Project managers can connect Helpdesk tickets to Project tasks. As the Helpdesk agent
+works on the ticket, they can use the ticket itself or the Project task to record timesheets.
+
+Thanks to the :guilabel:`Timesheet on Ticket` integration, timesheets will always be consistent and
+up-to-date on both records. Once a ticket is solved, the client can be billed for the time spent on
+the ticket. Odoo will pull from either the task or the ticket timesheet to accurately bill the
+client.
 
 Configuration
 =============
 
-| Go to :menuselection:`Helpdesk --> Configuration --> Helpdesk team --> Edit` and enable the options
-  *Timesheet on Ticket* and *Time Reinvoicing*.
-| Under *Timesheet on Ticket*, choose the *Project* to which tickets (and timesheets) will be
-  linked by default. Open its *External link* to enable the feature *Bill from tasks*.
+To turn on the :guilabel:`Timesheet on Ticket` feature, go to :menuselection:`Helpdesk -->
+Configuration --> Helpdesk Teams`, select a Helpdesk team, and then, click :guilabel:`Edit`. Next,
+enable the options :guilabel:`Timesheet on Ticket` and :guilabel:`Time Reinvoicing`.
+
+In the :guilabel:`Timesheet on Ticket` drop-down option, choose the Project to which tickets (and
+timesheets) will be linked by default. Then, open the Project's external link to edit its settings
+and enable the :guilabel:`Billable` option. Finally, save both the Project and the Helpdesk team's
+settings to apply these features.
+
+.. image:: reinvoice_from_project/project_billable.png
```

Comment:
I saw that @StraubCreative asked you to swap underscores for hyphens in #1982. If you want to do the same here, take the opportunity to change the commit tag ;)